code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -2562,11 +2562,12 @@
 	color = "#9A6750" //RGB: 154, 103, 80
 	taste_description = "inner peace"
 	penetrates_skin = NONE
+	var/datum/disease/transformation/gondolaDisease = /datum/disease/transformation/gondola
```

Comment:
```suggestion
	var/datum/disease/transformation/gondola_disease = /datum/disease/transformation/gondola
```

Reply:
I probably could have figured that out, done